### PR TITLE
Update kickstart.adoc and pom files

### PR DESCRIPTION
**Pull request by patriknw**
Refs #222

### CODE REVIEW

File: docs/src/modules/java/pages/kickstart.adoc [hunk 1]
```diff
@@ -133,7 +133,7 @@ For more details see xref:java:value-entity.adoc[].
 
 === Unit and integration tests
 
-The Akka Serverless Maven plugin creates a unit test stub for each class. Use this stub as a starting point to test the logic in your implementation. The Akka Serverless Java SDK test kit supports both link:https://junit.org/junit4/[JUnit 4] and link:https://junit.org/junit5/[JUnit 5].
+The Akka Serverless Maven plugin creates a unit test stub for Entity. Use this stub as a starting point to test the logic in your implementation. The Akka Serverless Java SDK test kit supports both link:https://junit.org/junit4/[JUnit 4] and link:https://junit.org/junit5/[JUnit 5].
```
Comment (johanandren) [inline docs/src/modules/java/pages/kickstart.adoc:136]:
```suggestion
The Akka Serverless Maven plugin creates a unit test stub for the Entity. Use this stub as a starting point to test the logic in your implementation. The Akka Serverless Java SDK test kit supports both link:https://junit.org/junit4/[JUnit 4] and link:https://junit.org/junit5/[JUnit 5].
```

**Review by johanandren — APPROVED** [review]

File: docs/src/modules/java/pages/kickstart.adoc (hunk 1 above)
Comment (octonato) [reply docs/src/modules/java/pages/kickstart.adoc:136]: We once decided to stick with junit5. 
See https://github.com/lightbend/akkaserverless-java-sdk/issues/41

I don't remember from where the idea comes from 'supporting'. Is it possible that the codegen was previously generating something for one or other style of tests? 

Anyway, I don't think we should change that in this PR, but we probably want to address #41 before we release a new SDK.

**Review by octonato — APPROVED** [review]
LGTM

File: maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/pom.xml [hunk 2]
```diff
@@ -299,18 +297,6 @@
...
-    <dependencies>
-      <dependency>
-        <groupId>com.typesafe.akka</groupId>
-        <artifactId>akka-bom_${scala.binary.version}</artifactId>
-        <version>${akka-streams.version}</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
```
Comment (raboof) [inline maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/pom.xml:312]: don't we want to promote this anymore?

**Review by raboof — APPROVED** [review]

File: maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/pom.xml (hunk 2 above)
Comment (patriknw) [reply maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/pom.xml:312]: We didn't have it in any of our samples. I don't know when or why it was introduced? It should work with transitive dependencies from sdk as long as user isn't starting to add Akka dependencies to their build.

Comment (raboof) [reply maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/pom.xml:312]: > I don't know when or why it was introduced?

Not sure either, looks like it's been there for ages (be8f8d6ea)

> It should work with transitive dependencies from sdk as long as user isn't starting to add Akka dependencies to their build.

This archetype *is* adding an Akka dependency to the build, though (akka-stream) - perhaps that can be removed?

Comment (patriknw) [reply maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/pom.xml:312]: This *is* the archetype and and I'm removing the akka-stream dependency here.

Comment (raboof) [reply maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/pom.xml:312]: > I'm removing the akka-stream dependency here.

Aaah gotcha I didn't put those together yet :+1:

**Pull request merged**